CMakeLists.txt

```diff
@@ -250,6 +250,7 @@ if(ENABLE_TRACY)
     include(${PROJECT_SOURCE_DIR}/cmake/tracy.cmake)
 endif()
 
+option(BUILD_PROGRAMMING_EXAMPLES "Enables build of tt_metal programming examples" OFF)
```

Comment:
Should we keep all `OPTION`s in `CMakeLists.txt` in one place ie. at top?

Reply:
I do like that idea.

Reply:
Addressed in commit https://github.com/tenstorrent/tt-metal/pull/13515/commits/a222f99ed5f24b1c7e61076e5f50911981903911